aws-ssm-maintenancewindowtarget/src/main/java/software/amazon/ssm/maintenancewindowtarget/translator/request/GetMaintenanceWindowTargetTranslator.java

```diff
@@ -0,0 +1,101 @@
+package software.amazon.ssm.maintenancewindowtarget.translator.request;
+
+import com.google.common.collect.ImmutableList;
+import software.amazon.awssdk.services.ssm.model.DescribeMaintenanceWindowTargetsRequest;
+import software.amazon.awssdk.services.ssm.model.DescribeMaintenanceWindowTargetsResponse;
+import software.amazon.awssdk.services.ssm.model.MaintenanceWindowTarget;
+import software.amazon.ssm.maintenancewindowtarget.MaintenanceWindowFilter;
+import software.amazon.ssm.maintenancewindowtarget.ResourceModel;
+import software.amazon.ssm.maintenancewindowtarget.translator.property.FiltersListTranslator;
+import software.amazon.ssm.maintenancewindowtarget.translator.property.TargetsListTranslator;
+import software.amazon.ssm.maintenancewindowtarget.util.SimpleTypeValidator;
+
+import java.util.Arrays;
+
+public class GetMaintenanceWindowTargetTranslator {
```

Comment:
Nit: might be better to call it DescribeMaintenanceWindowTargetTranslator?

Reply:
I named it with "Get..." to avoid confusion with the ListHandler, which is DescribeMaintenanceWindowTargets, although the ListHandler isn't implemented in the code.